tools/social-twitter/README.md

```diff
@@ -981,6 +1066,7 @@ The ID of the user to add to the list.
 
 The user was successfully added to the list.
 
+- \*\*`ok.is_member`
```

Comment:
Is this escaping necessary because of Gitbook?

Reply:
No, this looks like an error to me

Reply:
ref: https://github.com/Talus-Network/nexus-sdk/issues/176